packages/eslint-plugin/ROADMAP.md

```diff
@@ -51,6 +51,7 @@
 | [`import-blacklist`]                 | 🌟  | [`no-restricted-imports`][no-restricted-imports]                      |
 | [`label-position`]                   | 🌟  | [`no-unused-labels`][no-unused-labels] (similar)                      |
 | [`no-arg`]                           | 🌟  | [`no-caller`][no-caller] (also blocks `arguments.caller`)             |
+| [`no-async-without-await`]           | 🛑  | N/A                                                                   |
```

Comment:
this has an implementation in this plugin - `require-await`

Reply:
Oh, nice! Updated it.